internal/scorecard/alpha/tests/olm.go

```diff
@@ -326,14 +326,13 @@ func hasKind(kind1, kind2 string, r scapiv1alpha2.ScorecardTestResult) bool {
 	return strings.EqualFold(singularKind1, singularKind2)
 }
 
-func isCRFromCRDApi(cr unstructured.Unstructured, crds []*apiextv1.CustomResourceDefinition,
+func IsCRFromCRDApi(cr unstructured.Unstructured, crds []*apiextv1.CustomResourceDefinition,
```

Comment:
The capital letter allows us to use the method in other places. Is not this methods used here? So, if yes then the should not be changed.

Reply:
Sorry, I was thinking olm.go is in another package. Didn't realize both are under tests